packages/contentstack-import/src/import/modules-js/entries.js

```diff
@@ -486,7 +498,18 @@ module.exports = class ImportEntries {
           return resolve();
         })
         .catch((error) => {
-          addlogs(this.config, chalk.red("Failed to create entries in '" + lang + "' language"), 'error');
+          let title = JSON.parse(error?.request?.data ||"{}").entry?.title
+          addlogs(
+            this.config,
+            chalk.red(
+              "Failed to create entries in '" +
+                lang +
+                "' language. " +
+                'Title of the failed entry: ' +
+                title||"",
```

Comment:
@cs-raj Please use the quotes for the entity name for better readability.

Reply:
Done